GlobalExtensions.php

```diff
@@ -4,43 +4,29 @@
 wfLoadExtensions( [
 	'AbuseFilter',
 	'AntiSpoof',
-	'Babel',
-	'BetaFeatures',
```

Comment:
I'm not sure if we use BF globally in a way (I think we used to, unsure if we do now?)